src/jbi/model.py

```diff
@@ -0,0 +1,64 @@
+"""
+Python Module for Pydantic Models and validation
+"""
+import importlib
+from inspect import signature
+from types import ModuleType
+from typing import Any, Dict, Optional
+
+from pydantic import Extra, ValidationError, root_validator, validator
+from pydantic_yaml import YamlModel
+
+
+class ActionConfig(YamlModel, extra=Extra.allow):
+    """
+    ActionConfig is the inner model of `actions` in yaml
```

Comment:
This comment is a bit too similar to the one of `Actions`.

```suggestion
    ActionConfig is the inner model for each action in the configuration file
```